app/controllers/v0/messages_controller.rb

```diff
@@ -61,6 +51,22 @@ def thread
              meta: resource.metadata
     end
 
+    def reply
+      message = Message.new(message_params)
+
+      if message.body.blank?
+        message.errors.add(:body, "can't be blank")
+        raise Common::Exceptions::ValidationErrors, message
+      end
+
+      resource = client.post_create_message_reply(params[:id], message_params)
+      raise VA::API::Common::Exceptions::RecordNotFound, params[:id] unless resource.present?
```

Comment:
If you want you could try rescuing the specific ClientResponseError, but you would have to check the minor code that is returned and only handle that specific minor code by raising this exception. In short, this particular exception will never get raised so it can be removed.